### PR TITLE
update wget container for SRA_FASTQ_FTP from 1.20.1 to 1.21.4

### PR DESCRIPTION
**Pull request by ajandria**
## PR checklist

wget fails with `wget: unable to resolve host address 'ftp.sra.ebi.ac.uk'` as in #291.
This PR updates the container version for SRA_FASTQ_FTP from 1.20.1 to 1.21.4 which fixes this issue.

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/fetchngs/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, also make a PR on the nf-core/fetchngs _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ Thank you to everyone else that has contributed by reporting bugs, enhancements
 
 | Dependency | Old version | New version |
 | ---------- | ----------- | ----------- |
-| `wget`     |             | 1.20.1      |
+| `wget`     | 1.20.1      | 1.21.4      |
```
Comment (maxulysse) [inline CHANGELOG.md]: Can you add this part in the `Unpublished Version / DEV` section instead, this part is for the recent release

**Review by maxulysse — CHANGES_REQUESTED** [review]
Can you also add this in the CHANGELOG.
in the Enhancements & fixes section?

File: CHANGELOG.md (hunk 1 above)
Comment (ajandria) [reply CHANGELOG.md]: Ah, right. Done!

**Comment by ajandria** [comment]
@maxulysse Done!

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - [PR #299](https://github.com/nf-core/fetchngs/pull/299) - Template update for nf-core/tools v2.13.1
 - [PR #300](https://github.com/nf-core/fetchngs/pull/300) - Use file paths instead of tags for testing matrix, should make matrices more efficient
+- [PR #303](https://github.com/nf-core/fetchngs/pull/303) - Update wget container for SRA_FASTQ_FTP from 1.20.1 to 1.21.4
```
Comment (maxulysse) [inline CHANGELOG.md:12]:
```suggestion
- [PR #303](https://github.com/nf-core/fetchngs/pull/303) - Update wget container for SRA_FASTQ_FTP from 1.20.1 to 1.21.4

### Software dependencies

| Dependency | Old version | New version |
| ---------- | ----------- | ----------- |
| `wget`     | 1.20.1      | 1.21.4      |
```

**Review by maxulysse — APPROVED** [review]

**Pull request merged**